src/testUtils.tsx

```diff
@@ -155,18 +155,14 @@ const defaultUser = {
  *
  * Returns the new `axiosMock` in case you need to mock out axios requests.
  */
-export function initializeMocks({
-  user = defaultUser, initialState = undefined,
-  initializeStore = initializeReduxStore,
-}: {
+export function initializeMocks({ user = defaultUser, initialState = undefined }: {
```

Comment:
I reverted the changes to this file. It's better to specify the state in `editorRender`